.github/workflows/ci-testing.yml

```
@@ -47,7 +47,7 @@ jobs:
 
       - name: Tests
         working-directory: tests
-        run: pytest . -v --cov=litdata --durations=100
+        run: pytest . -v --cov=litdata --durations=100 -n auto --reruns 2
```

```suggestion
        run: pytest . -v --cov=litdata --durations=100 -n auto
```
rerun only specific tests, so in case something is really crashing, we won't take the test twice as long
```py
@pytest.mark.flaky(reruns=5)
```